incubator/hnc/docs/user-guide/how-to.md

```diff
@@ -328,6 +329,19 @@ Similarly, if you want to make `ns-bar` a root again, you must be an
 administrator of the root namespace that is an ancestor of `ns-bar`, since the
 admins of that namespace will lose access to `ns-bar` once it becomes a root.
 
+<a name="resolve-cond"/>
+
+### Resolve conditions on a namespace
+
+If the namespace has the following condition
+```
+ActivitiesHalted (ParentMissing): Parent "parent" does not exist
+```
+It means that this namespace is orphaned and its parent has been deleted. To fix this, you need to either create the parent, or mark this namespace as root namespace by using:
```

Comment:
Can you please wrap the lines to match the rest of this doc? I think it's either 80 or 100 columns.

Also: "mark this namespace as **a** root namespace:"

Reply:
done